packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -4,6 +4,11 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+
+/*
+ * Developer note! When updating this file, make sure to also update the
+ * _app-config template file in pwa-kit-create-app
```

Comment:
Let's be more detailed here and list all of the files because there is more than 1.

Reply:
In addition, can we also add a similar comment to the ssr.js file in template-retail-react-app?